scripts/publish.ps1

```diff
@@ -62,11 +62,17 @@ try {
     ce docker push "$Image"
 }
 catch {
-    Write-Output "`e[31m----------------------------------------------------------------`e[0m";
-    Write-Output "`e[31m Error occured in build and publishing image`e[0m";
-    Write-Output "`e[31m----------------------------------------------------------------`e[0m";
+    $local_error_message = @("`e[31m----------------------------------------------------------------`e[0m"
+                            ,"`e[31m Error occurred in build and publishing image`e[0m"
+                            ,"`e[31m----------------------------------------------------------------`e[0m"
+    );
+    Write-Output "`e[31m Error occurred in build and publishing image`e[0m"
 }
 finally {
+    if ($local_error_message)
+    {
+        Write-Error "`r`n$($local_error_message -join "`r`n")"
```

Comment:
Why do we have to do this change?  Entrypoint.sh of every action will anyway fail if any of the sub script is failing.

Reply:
The CI will succeed because we were only executing a `write-output` in the catch block. The build/publish fails but does not show in the workflow properly https://github.com/variant-inc/dx-demo-cron/runs/7138848856?check_suite_focus=true#step:5:792

Reply:
```powershell
$local_error_message = @"
`e[31m----------------------------------------------------------------`e[0m
`e[31m Error occurred in build and publishing image`e[0m
`e[31m----------------------------------------------------------------`e[0m
"@
```

Reply:
Change made as you suggested.